docs/source/kafka_check.rst

```diff
@@ -35,3 +35,37 @@ In case of min isr violations:
     CRITICAL: 1 partition(s) have the number of replicas in sync that is lower
     than the specified min ISR.
 
+Checking under replicated partitions
+====================================
+This kafka tool provides the ability to check number of under replicated partitions for all
+brokers in the cluster.
+
+The :code:`under_replicated` command checks if the number of under replicated partitions
+is equal to zero. This command will sum all numbers of under replicated
+partitions from each broker.
+
+The parameters for under_replicated check are:
+
+* :code:`--first-broker-only`: If this parameter is specified, it will do nothing and
+  succeed on not first broker in the list from Zookeeper. If :code:`--broker-id` is also
```

Comment:
I think you mean, 'The command will check for under-replicated partitions for given broker only if it's the first broker in broker-list fetched from zookeeper. Otherwise, it does nothing and succeeds.